docs/content/docs/tutorials/aurora-serverless-v2.md

```diff
@@ -0,0 +1,341 @@
+
+1. Create a [workspace](https://www.eksworkshop.com/020_prerequisites/workspace/)
+2. Install the [Kubernetes tools](https://www.eksworkshop.com/020_prerequisites/k8stools/)
+3. Create an [IAM role for workspace](https://www.eksworkshop.com/020_prerequisites/iamrole/)
+4. Attach [IAM role to workspace](https://www.eksworkshop.com/020_prerequisites/ec2instance/)
+5. Update [IAM settings](https://www.eksworkshop.com/020_prerequisites/workspaceiam/)
+6. [Create KMS custom managed keys](https://www.eksworkshop.com/020_prerequisites/kmskey/)
+
+### Method 2: Deploy an Amazon EKS cluster
+
+1. Create an Amazon EKS cluster
+2. Install [eksctl tools](https://www.eksworkshop.com/030_eksctl/prerequisites/)
+3. Install [HELM](https://www.eksworkshop.com/beginner/060_helm/helm_intro/install/)
+4. Install [EKS](https://www.eksworkshop.com/030_eksctl/launcheks/)
+5. Test [EKS cluster](https://www.eksworkshop.com/030_eksctl/test/)
+6. (Optional) [Grant console access to EKS cluster](https://www.eksworkshop.com/030_eksctl/console/#:~:text=The%20EKS%20console%20allows%20you,granted%20permission%20within%20the%20cluster.)
+
+## Install ACK service controller for RDS
+
+To manage an Aurora Serverless v2 cluster from Kubernetes / Amazon EKS, you will
+need to install the ACK for RDS service controller. The commands below download
+the latest Helm chart for the ACK for RDS service controller and prepare it to
+be installed in your environment.
+
+```bash
+export HELM_EXPERIMENTAL_OCI=1
+export SERVICE=rds
+export RELEASE_VERSION=$(curl -sL https://api.github.com/repos/aws-controllers-k8s/rds-controller/releases/latest | grep '"tag_name":' | cut -d'"' -f4)
+export CHART_EXPORT_PATH=/tmp/chart
+export CHART_REF="${SERVICE}-chart"
+export CHART_REPO="public.ecr.aws/aws-controllers-k8s/${CHART_REF}"
+export CHART_PACKAGE="${CHART_REF}-${RELEASE_VERSION}.tgz"
+
+mkdir -p "${CHART_EXPORT_PATH}"
+
+export ACK_SYSTEM_NAMESPACE=ack-system
+```
+
+Deploy the ACK service controller for Amazon RDS using the [Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/rds-chart). You can download it to your
+workspace using the following commands:
+
+```bash
+aws ecr-public get-login-password --region us-east-1 | helm registry login --username AWS --password-stdin public.ecr.aws
+helm pull "oci://${CHART_REPO}" --version "${RELEASE_VERSION}" -d "${CHART_EXPORT_PATH}"
+```
+
+Decompress and extract the Helm chart:
+
+```bash
+tar xvf "${CHART_EXPORT_PATH}/${CHART_PACKAGE}" -C "${CHART_EXPORT_PATH}"
+```
+
+You can now use the Helm chart to deploy the ACK service controller for
+Amazon RDS to your Amazon EKS cluster. At a minimum, you need to specify the
+AWS Region to execute the RDS API calls.
+
+```bash
+helm install --create-namespace \
+  --namespace "${ACK_SYSTEM_NAMESPACE}"  \
+  --set aws.region="$AWS_REGION" \
+  "ack-${SERVICE}-controller" "${CHART_EXPORT_PATH}/${SERVICE}-chart"
+
+helm list --namespace "${ACK_SYSTEM_NAMESPACE}" -o yaml    
+```
```

Comment:
This methodology is outdated. You can see how we achieve this using Helm 1.8+ compatible commands from our existing tutorial: https://aws-controllers-k8s.github.io/community/docs/tutorials/apigatewayv2-reference-example/#install-the-ack-service-controller-for-apigatewayv2

Reply:
:+1: This was borrowed from the RDS example, but it looks like that was updated while this tutorial was being written (which makes my "markdown-rot" comment hold :wink:). I'll update this with the new methodology.